src/analytics/navigationTracker/navigationTracker.ts

```diff
@@ -183,4 +213,65 @@ export class NavigationTracker extends BaseTracker {
     const elementClassName: string = element.className as string;
     return elementClassName.includes("govuk-back-link");
   }
+
+  /**
+   * Determines if the given class name is a logo link
+   *
+   * @param {string} element - The HTML link element to get the type of.
+   * @return {boolean} Returns true if the class name of this element includes "govuk-header__logo", false otherwise.
+   */
+  isLogoLink(element: HTMLElement): boolean {
+    const logo = document.getElementsByClassName("govuk-header__logo")[0];
+    return logo && logo.contains(element);
+  }
+
+  /**
+   * Determines if the given tagname is a is a nav link
+   *
+   * @param {string} element - The HTML link element to get the type of.
+   * @return {boolean} Returns true if the nav tag contains this element, false otherwise.
+   */
+  isNavLink(element: HTMLElement): boolean {
```

Comment:
I'm not sure about this one.
If I'm testing your code against HRMC website, and if you click on MENU, the links are not contained in a nav tag.
https://www.gov.uk/government/organisations/hm-revenue-customs
I think we need to add multiple of check if we want to know it's a nav link.

Reply:
I think the HRMC website menu links are within the header which is part of [the header component](https://design-system.service.gov.uk/components/header/), which is on the account repo aswell (i searched 'signout'. I left a note above about the example used in the briefing document by Sam's team. The only example I can find that matches is on the design system is on the GOV design system site, which uses the 'nav' tag.... https://design-system.service.gov.uk/components/

Reply:
@di-fabs do you think i should like this to the header's nav links perhaps instead of the tag name <nav>